go/vt/vtgate/vtgate.go

```diff
@@ -132,6 +134,7 @@ func registerFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&enableSchemaChangeSignal, "schema_change_signal", enableSchemaChangeSignal, "Enable the schema tracker; requires queryserver-config-schema-change-signal to be enabled on the underlying vttablets for this to work")
 	fs.StringVar(&schemaChangeUser, "schema_change_signal_user", schemaChangeUser, "User to be used to send down query to vttablet to retrieve schema changes")
 	fs.IntVar(&queryTimeout, "query-timeout", queryTimeout, "Sets the default query timeout (in ms). Can be overridden by session variable (query_timeout) or comment directive (QUERY_TIMEOUT_MS)")
+	fs.BoolVar(&vstreamCellAliasFallback, "vstream_cell_alias_fallback", vstreamCellAliasFallback, "Determines whether to default to cell aliases for VStream tablet selection instead of having the gRPC request specify this in the VStreamrRequest object.")
```

Comment:
We're moving from underscores to dashes so we should use dashes for all new flags. I think the description would be a little more clear if it was ~:
```
fs.BoolVar(&vstreamUseCellAlias, "vstream-use-cell-alias", vstreamUseCellAlias, "By default, use a cell alias which contains the vtgate's local cell when selecting tablets for streaming")
```

Since I think the default, when no specific behavior was selected in the gRPC call, goes from `search for tablets in the local cell` to `search for tables in a cell alias that contains the local cell or if there isn't one, then search for tablets in the local cell`. No?